examples/timit/asr1/README.md

```diff
@@ -0,0 +1,195 @@
+# Transformer/Conformer ASR with Timit
+The phoneme-based continuous speech corpus is a collaboration between Texas Instruments, MIT, and SRI International. The [Timit](https://catalog.ldc.upenn.edu/docs/LDC93S1/) dataset has a voice sampling frequency of 16 khz and contains a total of 6,300 sentences, with 630 people from eight major U.S. dialects speaking a given 10 sentences each, all sentences are manually segmented and marked at the phone level. Seventy percent of the speakers are male; most of the speakers are white adults.
+
+## Dataset
+### Download and Extract
+Download TIMIT from it's [official website](https://catalog.ldc.upenn.edu/LDC93S1) and extract it to `~/datasets`. Then the dataset is in the directory `~/datasets/timit`.
```

Comment:
data directory should same with `run.sh`.

Reply:
I've not unzip the TIMIT.zip to the same path with run.sh.
I've unzip the TIMIT.zip to the datasets.